prime-router/setup_environment.sh

```diff
@@ -0,0 +1,325 @@
+storage_key=$(az storage account keys list --account-name $storage_account \
+                                           --output tsv \
+                                           --query [0].value)
+
+az storage share-rm create --resource-group $resource_group \
+                           --name $sftp_server_name \
+                           --storage-account $storage_account \
+                           --output none \
+                           --only-show-errors
+
+echo "--- Creating SFTP server container..."
+az container create --resource-group $resource_group \
+                    --name $sftp_server_name \
+                    --image atmoz/sftp:latest \
+                    --vnet $vnet_name \
+                    --subnet container \
+                    --ports 22 \
+                    --environment-variables SFTP_USERS=foo:pass:::upload \
+                    --azure-file-volume-share-name $sftp_server_name \
+                    --azure-file-volume-account-name $storage_account \
+                    --azure-file-volume-account-key $storage_key \
+                    --azure-file-volume-mount-path /home/foo/upload \
+                    --output none \
+                    --only-show-errors
+
+echo "--- Creating front door..."
+az network front-door create --resource-group $resource_group \
+                             --name $front_door \
+                             --backend-address $function_app.azurewebsites.net \
+                             --accepted-protocols Https \
+                             --output none \
+                             --only-show-errors
+
+echo -e "\t- Applying access restrictions..."
+# IMPORTANT - The above access restriction will prevent open access
+# to the function app.  However, an access restriction must be manually
+# created to allow traffic via FrontDoor.Backend.  This cannot currently
+# be done via CLI as it is in preview.
+
+az functionapp config access-restriction set --resource-group $resource_group \
+                                             --name $function_app \
+                                             --use-same-restrictions-for-scm-site true \
+                                             --output none \
+                                             --only-show-errors
+
+az functionapp config access-restriction add --resource-group $resource_group \
+                                             --name $function_app \
+                                             --rule-name AllowVNetTraffic \
+                                             --priority 100 \
+                                             --action Allow \
+                                             --vnet-name $vnet_name \
+                                             --subnet public \
+                                             --output none \
+                                             --only-show-errors
+
+# IMPORTANT - NEED TO SET SFTP CONTAINER ENVIRONMENT VARIABLES!!
```

Comment:
and please also add 
      - AZ_PHD__ELR_HL7_TEST__USER=****
      - AZ_PHD__ELR_HL7_TEST__PASS=****

confusingly, despite the "TEST" in the name, this does get added to both the Test and Production envs. In test with Test creds, and in with the Production with the real connection creds.